src/extensions/default/bramble/lib/ConsoleInterfaceManager.js

```diff
@@ -0,0 +1,137 @@
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        Mustache = brackets.getModule("thirdparty/mustache/mustache"),
+        Strings = brackets.getModule("strings");
+
+    var panel,
+        icon = null,
+        panelHTML = require("text!htmlContent/console.html");
+    
+    // panelHTML = Mustache.parse(panelHTML, {"Strings": Strings});
+
+    icon = $("<button class=\"editor-console-icon-indicator\" title='{{CONSOLE_TOOLTIP}}'>{{CONSOLE_TITLE}}</button>");
+    // icon = Mustache.parse(icon, {"Strings": Strings});
+    icon.appendTo($("#editor-holder"));
+
+    var logData = [],
+        livePreviewOnly = false;
+
+    var wasClosedByUser = false;
+
+    function count(data, type) {
+        var i = 0,
+            result = 0;
+
+        for (i = 0; i < data.length; i++) {
+            if (data[i].type === type && (!livePreviewOnly)) {
+                result++;
+            }
+        }
+        return result;
+    }
+
+    function clear() {
+        var $console = panel.$panel.find(".console");
+        $console.html("");
+    }
+
+    function render() {
+        var $console = panel.$panel.find(".console"),
+            $element = "", 
+            data = logData, 
+            i = 0;
+
+        clear();
+
+        // This creates the individual messages
+        for (i = 0; i < data.length; i++) {
+            $element = $("<div class='" + data[i].type + "'>" + data[i].text +"</div>");
+            $console.append($element);
+        }
+
+        $console.animate({ scrollTop: $console[0].scrollHeight }, 10);
+    }
+
+    function add(type, args) {
+        // Display the console when user code triggers console.* functions
+        if(!panel.isVisible() && !wasClosedByUser) {
+            panel.show();
+        }
+        
+        var texts = args.toString().split('\n'), 
+            i = 0;
+
+        for (i = 0; i < texts.length; i++) {
+            logData.push({type: type, text: texts[i]});
+        }
+
+        render();
+    }
+
+    function togglePanel() {
+        if (panel.isVisible()) {
+            panel.hide();
+            icon.removeClass("on");
+        } else {
+            panel.show();
+            wasClosedByUser = false;
+            icon.addClass("on");
+        }
+    }
+
+    AppInit.htmlReady(function () {
+
+        ExtensionUtils.loadStyleSheet(module, "../stylesheets/consoleTheme.css");
+
+        panel = WorkspaceManager.createBottomPanel("console.panel", $(panelHTML));
+
+        panel.$panel.find("#btnClear").on("click", function () {
+            clear();
+        });
+
+        panel.$panel.find(".close").on("click", function () {
+            panel.hide();
+            icon.removeClass("on");
+            wasClosedByUser = true;
+        });
+
+        icon.on("click", togglePanel);
+
+        // Localization
+        panelHTML = Mustache.render(panelHTML, {"Strings": Strings});
+        icon = Mustache.render(icon, {"Strings": Strings});
```

Comment:
You cannot pass anything other than a string to `Mustache.render` as the first param. In this case, `icon` is a jquery object.